src/components/Tooltip/index.tsx

```diff
@@ -25,25 +29,26 @@ interface TooltipProps extends Omit<PopoverProps, 'content'> {
   text: ReactNode
   open?: () => void
   close?: () => void
-  noOp?: () => void
   disableHover?: boolean // disable the hover and content display
   timeout?: number
 }
 
 interface TooltipContentProps extends Omit<PopoverProps, 'content'> {
   content: ReactNode
   onOpen?: () => void
+  open?: () => void
```

Comment:
How is this different from `onOpen`? Why aren't these labeled as callbacks (eg `onOpen`/`onClose`)?